src/Goerli-DssSpell.t.sol

```diff
@@ -466,31 +464,59 @@ contract DssSpellTest is GoerliDssSpellTestBase {
     }
 
     function testVestDAI() private { // make public to use
-        VestAbstract vest = VestAbstract(addr.addr("MCD_VEST_DAI"));
+        // VestAbstract vest = VestAbstract(addr.addr("MCD_VEST_DAI"));
 
-        assertEq(vest.ids(), 0);
+        // assertEq(vest.ids(), 0);
 
-        vote(address(spell));
-        scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done());
+        // vote(address(spell));
+        // scheduleWaitAndCast(address(spell));
+        // assertTrue(spell.done());
 
-        assertEq(vest.ids(), 1);
+        // assertEq(vest.ids(), 1);
 
-        assertEq(vest.cap(), 1 * MILLION * WAD / 30 days);
+        // assertEq(vest.cap(), 1 * MILLION * WAD / 30 days);
 
-        assertEq(vest.usr(1), address(pauseProxy));
-        assertEq(vest.bgn(1), block.timestamp - 1 days);
-        assertEq(vest.clf(1), block.timestamp - 1 days);
-        assertEq(vest.fin(1), block.timestamp);
-        assertEq(vest.mgr(1), address(0));
-        assertEq(vest.res(1), 0);
-        assertEq(vest.tot(1), WAD);
-        assertEq(vest.rxd(1), 0);
+        // assertEq(vest.usr(1), address(pauseProxy));
+        // assertEq(vest.bgn(1), block.timestamp - 1 days);
+        // assertEq(vest.clf(1), block.timestamp - 1 days);
+        // assertEq(vest.fin(1), block.timestamp);
+        // assertEq(vest.mgr(1), address(0));
+        // assertEq(vest.res(1), 0);
+        // assertEq(vest.tot(1), WAD);
+        // assertEq(vest.rxd(1), 0);
 
-        uint256 prevBalance = dai.balanceOf(address(pauseProxy));
-        assertTrue(tryVest(address(vest), 1));
-        assertEq(dai.balanceOf(address(pauseProxy)), prevBalance + WAD);
+        // uint256 prevBalance = dai.balanceOf(address(pauseProxy));
+        // assertTrue(tryVest(address(vest), 1));
+        // assertEq(dai.balanceOf(address(pauseProxy)), prevBalance + WAD);
 
-        assertEq(vest.rxd(1), WAD);
+        // assertEq(vest.rxd(1), WAD);
     }
+
+   function testTeleportFW() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        TeleportRouterLike router = TeleportRouterLike(addr.addr("MCD_ROUTER_TELEPORT_FW_A"));
+        StarknetTeleportBridgeLike bridge = StarknetTeleportBridgeLike(addr.addr("STARKNET_TELEPORT_BRIDGE"));
+
+       // Sanity check
+        assertEq(router.numDomains(), 4);
+
+       checkTeleportFWIntegration(
+            "STA-GOER-A",
+            "ETH-GOER-A",
+            100_000 * WAD,
+            address(bridge),
+            addr.addr("STARKNET_TELEPORT_FEE"),
+            addr.addr("STARKNET_ESCROW"),
+            100 * WAD,
+            WAD / 10000, // 1bps
+            30 minutes
+        );
+
+       // Bridge domain specific checks
+       assertEq(bridge.l2TeleportGateway(), 0x078e1e7cc88114fe71be7433d1323782b4586c532a1868f072fc44ce9abf6714);
+       assertEq(bridge.starkNet(), 0xde29d060D45901Fb19ED6C6e959EB22d8626708e);
```

Comment:
There is a key in the addrs file to check this, anyway this is explicitly check in the spell so could also be removed from here. Same for the prev check.

Reply:
Changed the 2nd line to use the address file.
1st line is not there, so currently following the pattern of the 30/8 spell to use specific address.

Although these checks are also in the spell I think it's good to have them here as someone might remove them from the spell on future onboardings. We are checking the other ctr params in the base test so might as well check the unique params here.